package/src/lib/SpotlightTour.context.ts

```diff
@@ -186,6 +175,14 @@ export interface SpotlightTourCtx extends SpotlightTour {
    * @param spot the spot layout
    */
   changeSpot: (spot: LayoutRectangle) => void;
+  /**
+   * Specifies {@link FloatingProps} in order to configure Floating UI
+   * in all tour steps layout.
+   *
+   * @default middlewares: [flip(), offset(4), shift()]
+   * @default placement: "bottom"
+   */
+  floatingUiConfigurations: FloatingProps;
```

Comment:
We can simplify this name. What do you think about keeping it as `floatingProps` for consistency with the type? 🙂

Reply:
I dont think we can use floatingProps in the spotlight tour context because the component already receives a floatingProps object configuration this means we will have 2 different variables under the same scope with same name

Reply:
I dont think we can use floatingProps in the spotlight tour context because the component already receives a floatingProps object configuration this means we will have 2 different variables under the same scope with same name